src/js/containers/Buttons.js

```diff
@@ -42,7 +42,7 @@ const mapStateToProps = (state) => {
             icon: iconSeekLeft
         })
     }
-    if (subscribedButtons.OK === true && app.updateMode === "PAUSE") {
+    if (subscribedButtons.OK === true && (app.audioStreamingIndicator === "PLAY" || app.updateMode === "PAUSE")) {
```

Comment:
audioStreamingIndicator should take precedence if it is present, updateMode should only be used if it isn't set or is `PLAY_PAUSE`.